qdax/baselines/sac_pbt.py

```diff
@@ -0,0 +1,455 @@
+"""
+A collection of functions and classes that define the algorithm Soft Actor Critic
```

Comment:
A few docstrings in this file are not up to date. They are mentioning SAC instead of PBT-SAC.

Reply:
done